mcsas3_cli_runner.py

```diff
@@ -131,7 +132,7 @@ def isWindows():
         default=Path(__file__).absolute().parent
         / "example_configurations"
         / "run_config_spheres_auto.yaml",
-        help="Path to the filename with the SAXS data",
+        help="Path to the config how with the MCS/fitting parameters",
```

Comment:
Would rename to "Path to the configuration file containing the model parameters"